### PR TITLE
Jaeger configuration and deployment options (#20643 restored)

### PR DESCRIPTION
**Pull request by JStickler**
I had two PRs open in different tabs and merged  https://github.com/openshift/openshift-docs/pull/20643  by accident while I was trying to troubleshoot build errors.  This PR contains all the changes from that PR.

### CODE REVIEW

**Comment by objectiser** [comment]
@JStickler On re-ready the PR, do you think we need a more explicit statement that Jaeger has been tested (and is supported) with AMQ streams, rather than other kafka products?

**Comment by JStickler** [comment]
@objectiser, rather than hold up this PR, why don't we create an issue so that we can work out what to say (and where to put the content, since we don't have a "supported configurations" topic yet, and should probably add one).

**Review by objectiser — APPROVED** [review]

**Comment by objectiser** [comment]
@JStickler Created https://issues.redhat.com/browse/TRACING-1212 to track that. Added to sprint #41 but feel free to schedule in whichever sprint you think appropriate.

**Review by kevinearls — APPROVED** [review]

File: pipelines/op-release-notes.adoc
Comment (neal-timpe) [inline pipelines/op-release-notes.adoc:31]: Is this related?

File: modules/jaeger-config-collector.adoc
Comment (neal-timpe) [inline modules/jaeger-config-collector.adoc:13]: I think in this table, if there are no default values, we should remove that column.

**Review by neal-timpe — CHANGES_REQUESTED** [review]
I think there are a couple of items that need review before merging.

File: installing/installing_azure/installing-azure-user-infra.adoc
Comment (JStickler) [reply installing/installing_azure/installing-azure-user-infra.adoc]: This was leftover from my attempts to debug build errors.  I've reverted to the file from master.

File: installing/install_config/installation-types.adoc
Comment (JStickler) [reply installing/install_config/installation-types.adoc]: Same, this was an attempt at debugging build errors.  Reverted this file to the version on the master branch as well.

File: pipelines/op-release-notes.adoc
Comment (JStickler) [reply pipelines/op-release-notes.adoc:31]: I was getting build errors because immediately after this someone opened a multi-line comment and didn't close it with four slashes. I thought separating that chuck of text from the includes comment might help avoid that error getting re-introduced.

File: modules/jaeger-config-collector.adoc
Comment (JStickler) [reply modules/jaeger-config-collector.adoc:13]: Done.

**Review by neal-timpe — APPROVED** [review]

**Pull request merged**
